tests_e2e/test_suites/images.yml

```diff
@@ -46,6 +46,30 @@ image-sets:
       - "ubuntu_1804"
       - "ubuntu_2004"
 
+   # agent persist firewall service not running on flatcar distro since agent can't install custom service due to read only filesystem.
```

Comment:
does this mean the agent is currently failing in flatcar?

Reply:
no, somehow agent is working but getting error while creating service unit file in /lib/systemd/system. They must have setup only read permissions.

Reply:
ok

Reply:
creating service, no firewalld service running

Reply:
ok, so the agent can create the service, and the test can't? why?

Reply:
oh, i misread the comment, i guess

but why skip flatcar altogether?

Reply:
oh we are talking different things. My test won't create any service file. My test checks service is enabled by the agent. Agent not able to crate this file because distro is not allowing it. 

Means, we don't have persist firewall setup in this distro

Reply:
2023-11-14T19:04:13.728927Z INFO ExtHandler ExtHandler Starting setup for Persistent firewall rules
2023-11-14T19:04:13.731799Z INFO ExtHandler ExtHandler Firewalld service not running/unavailable, trying to set up waagent-network-setup.service
2023-11-14T19:04:13.732237Z INFO ExtHandler ExtHandler Successfully updated the Binary file /var/lib/waagent/waagent-network-setup.py for firewall setup
2023-11-14T19:04:13.738256Z INFO ExtHandler ExtHandler Service: waagent-network-setup.service not enabled. Adding it now
2023-11-14T19:04:13.738695Z ERROR ExtHandler ExtHandler Unable to setup the persistent firewall rules: [Errno 30] Read-only file system: '/lib/systemd/system/waagent-network-setup.service

Reply:
`but why skip flatcar altogether?
`

No persist firewall setup, so nothing to validate.

Reply:
so coming back to my original question: the agent is failing on flatcar, right? the above log seems to indicate so

Reply:
I misread your question first place then, sorry for the confusion. I thought you ask agent totally broken. Yes, agent failing in firewall setup.

Reply:
It's ok. I was not aware of this, or maybe at some point I was and I forgot about it. Could you open a work item for this?

When you finish skipping Flatcar, please add the above error as a comment to explain why we are skipping